src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -80,11 +82,55 @@ def remove_at_portal_types(tool):
     logger.info("Remove AT types from portal_types tool ... [DONE]")
 
 
+def migrate_to_dx(portal_type, origin, dest, schema_mapping):
+    """Migrates Setup AT contents to Dexterity
+    """
+    logger.info("Migrating {} to Dexterity ...".format(portal_type))
+
+    # copy items from old -> new container
+    objects = origin.objectValues()
+    for src in objects:
+        if api.get_portal_type(src) != portal_type:
+            logger.error("Not a '{}' object: {}".format(portal_type, src))
+            continue
+
+        # Create the object if it does not exist yet
+        src_id = src.getId()
+        target = dest.get(src_id)
+        if not target:
+            # Don' use the api to skip the auto-id generation
+            target = createContent(portal_type, id=src_id)
+            dest._setObject(src_id, target)
+            target = dest._getOb(src_id)
+
+        # Migrate the contents from AT to DX
+        migrator = getMultiAdapter(
+            (src, target), interface=IContentMigrator)
+        migrator.migrate(schema_mapping, delete_src=False)
+
+        # Delete the old AT object
+        delete_object(src)
```

Comment:
Please remove this line and pass in `delete_src=True` to the migrator call above